File: python/Ejercicio22.py
import math

x1 = float(input("Ingrese la coordenada X del primer punto: "))
y1 = float(input("Ingrese la coordenada Y del primer punto: "))
x2 = float(input("Ingrese la coordenada X del segundo punto: "))
y2 = float(input("Ingrese la coordenada Y del segundo punto: "))

distancia = math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)

print("La distancia entre los dos puntos es:", distancia)

File: java/Ejercicio22.java
import java.util.Scanner;

public class Ejercicio22 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        

        System.out.print("Ingrese la coordenada X del primer punto: ");
        double x1 = sc.nextDouble();

        System.out.print("Ingrese la coordenada Y del primer punto: ");
        double y1 = sc.nextDouble();

        System.out.print("Ingrese la coordenada X del segundo punto: ");
        double x2 = sc.nextDouble();

        System.out.print("Ingrese la coordenada Y del segundo punto: ");
        double y2 = sc.nextDouble();

        double distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

        System.out.println("La distancia entre los dos puntos es: " + distancia);

        sc.close();
    }
}